src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,33 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+  const commentExists = comments.length !== 0;
```

Comment:
```suggestion
  const hasComments = comments.length !== 0;
```
Boolean variables should start from `is` or `has`